app/res/xml/preferences_developer.xml

```diff
@@ -85,6 +85,11 @@
         android:entryValues="@array/pref_enabled_vals"
         android:key="cc-auto-purge"
         android:title="Detect and Trigger Purge on Form Save"/>
+    <EditTextPreference
```

Comment:
So I took a look at the .ccz installation code. Actually for .ccz installation we have both options , to write a file path manually and a  folder icon to choose it from file manager. Calabash tests never actually utilise the file manager option and directly put the path in EditText. I am going to try to do the same here and build a custom preference FilePreference for such prefs.